instrumentation/github.com/gocql/gocql/otelgocql/test/gocql_test.go

```diff
@@ -110,70 +108,70 @@ func TestQuery(t *testing.T) {
 		assertConnectionLevelAttributes(t, span)
 	}
 
-	// Check metrics
-	actual := obtainTestRecords(meterProvider.MeasurementBatches)
-	require.Len(t, actual, 3)
-	expected := []testRecord{
-		{
-			name:      "db.cassandra.queries",
-			meterName: internal.InstrumentationName,
-			attributes: []attribute.KeyValue{
-				internal.CassDBSystem(),
-				internal.CassPeerIP("127.0.0.1"),
-				internal.CassPeerPort(9042),
-				internal.CassVersion("3"),
-				internal.CassHostID("test-id"),
-				internal.CassHostState("UP"),
-				internal.CassKeyspace(keyspace),
-				internal.CassStatement(insertStmt),
-			},
-			number: 1,
-		},
-		{
-			name:      "db.cassandra.rows",
-			meterName: internal.InstrumentationName,
-			attributes: []attribute.KeyValue{
-				internal.CassDBSystem(),
-				internal.CassPeerIP("127.0.0.1"),
-				internal.CassPeerPort(9042),
-				internal.CassVersion("3"),
-				internal.CassHostID("test-id"),
-				internal.CassHostState("UP"),
-				internal.CassKeyspace(keyspace),
-			},
-			number: 0,
-		},
-		{
-			name:      "db.cassandra.latency",
-			meterName: internal.InstrumentationName,
-			attributes: []attribute.KeyValue{
-				internal.CassDBSystem(),
-				internal.CassPeerIP("127.0.0.1"),
-				internal.CassPeerPort(9042),
-				internal.CassVersion("3"),
-				internal.CassHostID("test-id"),
-				internal.CassHostState("UP"),
-				internal.CassKeyspace(keyspace),
-			},
-		},
-	}
-
-	for _, record := range actual {
-		switch record.name {
-		case "db.cassandra.queries":
-			recordEqual(t, expected[0], record)
-			assert.Equal(t, expected[0].number, record.number)
-		case "db.cassandra.rows":
-			recordEqual(t, expected[1], record)
-			assert.Equal(t, expected[1].number, record.number)
-		case "db.cassandra.latency":
-			recordEqual(t, expected[2], record)
-			// The latency will vary, so just check that it exists
-			assert.True(t, !record.number.IsZero(record.numberKind))
-		default:
-			t.Fatalf("wrong metric %s", record.name)
-		}
-	}
+	// // Check metrics
```

Comment:
Is this also being replaced with the in memory exporter from https://github.com/open-telemetry/opentelemetry-go/issues/2722?

Reply:
Yes.
```suggestion
	// TODO: Replace with in memory exporter https://github.com/open-telemetry/opentelemetry-go/issues/2722
	// // Check metrics
```